src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/ScenarioTests/AzureSql/ItemTests.ps1

```diff
@@ -42,47 +43,48 @@ function Test-AzureSqlGetItems
 
 function Test-AzureSqlDisableProtection
 {
-	#1. Get the vault
+	# 1. Get the vault
 	$vault = Get-AzureRmRecoveryServicesVault -ResourceGroupName "sqlpaasrg" -Name "sqlpaasrn";
 	
-	# 2. Set the vault context
-	Set-AzureRmRecoveryServicesVaultContext -Vault $vault;
-	
-	# 3. Get the container
+	# 2. Get the container
 	$namedContainer = Get-AzureRmRecoveryServicesBackupContainer `
+		-Vault $vault `
 		-ContainerType "AzureSQL" `
 		-BackupManagementType "AzureSQL" `
 		-Name "Sql;sqlpaasrg;sqlpaasserver";
 	Assert-AreEqual $namedContainer.Name "Sql;sqlpaasrg;sqlpaasserver";
 
-	# 4. Get named item for container
+	# 3. Get named item for container
 	$item = Get-AzureRmRecoveryServicesBackupItem `
+		-Vault $vault `
 		-Container $namedContainer `
 		-WorkloadType "AzureSQLDatabase" `
 		-Name "dsName;satyay-sea-d1-fc1-catalog-2016-11-11-17-20;661f0942-d5b7-486a-b3cb-68f97d325a3c";
 	Assert-AreEqual $item.Name "dsName;satyay-sea-d1-fc1-catalog-2016-11-11-17-20;661f0942-d5b7-486a-b3cb-68f97d325a3c";
 
 	$job = Disable-AzureRmRecoveryServicesBackupProtection `
-		-Item $item -RemoveRecoveryPoints -Force;
+		-Vault $vault `
+		-Item $item `
+		-RemoveRecoveryPoints `
+		-Force;
 }
 
 function Test-AzureSqlGetRPs
 {
-	#Set vault context
+	# 1. Get the vault
 	$vault = Get-AzureRmRecoveryServicesVault -ResourceGroupName "sqlpaasrg" -Name "sqlpaasrn";
 	
-	# 2. Set the vault context
```

Comment:
Can you have one test where you set the context with Set-AzureRmRecoveryServicesVaultContext? I assume we want this to continue to work, so we should have at least one test for it.

Reply:
Done.